components/NotificationPopup/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { useState } from "react";
+
+import Image from "next/image";
+
+import classNames from "classnames/bind";
+
+import { useGetAllNotification } from "@/hooks/useGetAllNotification";
+
+import styles from "./NotificationPopup.module.scss";
+import BaseNotification from "../Notification/BaseNotification";
+import EmptyNotification from "../Notification/EmptyNotification";
+
+const cn = classNames.bind(styles);
+
+export default function NotificationPopup() {
+  const [isClicked, setIsClicked] = useState(false);
+
+  const { data: notificationInfo } = useGetAllNotification();
+
+  const getNotificationCount = () => {
+    if (!notificationInfo) return;
+
+    return notificationInfo.count < 10 ? notificationInfo.count : "+";
+  };
+
+  const handleBellClick = () => {
+    setIsClicked(!isClicked);
+  };
+
+  return (
+    <div className={cn("container")}>
+      <figure onClick={handleBellClick} className={cn("notification-bell")}>
+        {notificationInfo?.count && (
+          <figcaption className={cn("notification-count")}>{getNotificationCount()}</figcaption>
+        )}
+        <Image src="/icons/notification-bell.svg" width={26} height={20} alt="알림 종" priority />
+      </figure>
+      {isClicked && (
+        <div className={cn("notification-popup")}>
+          {notificationInfo?.crewNotificationResponse.length ? (
```

Comment:
```suggestion
          {notificationList.length > 0 (
```
P4: 이 부분은 이름이 너무 길어서 위에서 `notificationInfo`을 한번 분해해서 사용하면 가독성 향상에 도움이 될 것 같아요!
```js
const { count, crewNotificationResponse: notificationList } = notificationInfo
```
그리고 추가적으로 저는 `notificationList.length > 0` 이렇게 사용하는게 좀 더 명시적이고 완전한 조건식을 제공해서 이 방식을 선호합니다!